src/editor.jsx

```diff
@@ -565,9 +565,6 @@ var Editor = React.createClass({
 
         localStorage.perseusLastCopiedText = selectedText;
         localStorage.perseusLastCopiedWidgets = JSON.stringify(widgetData);
-
-        console.log(
```

Comment:
iirc widget copying was implemented as an intern project and i'm 90% sure this is just for debugging. it's maybe nice if a dev wants to inspect what happened on the perseus clipboard in the terminal, but I'd bet we don't actually do that.

I think it's okay to get rid of this.